lib/workarounds/bugzilla_issues.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+package bugzilla_issues;
+
+use testapi;
+use utils;
+use warnings;
+use strict;
+use Exporter;
+our @EXPORT_OK = qw(
+  workaround_1175335
+);
+
+sub workaround_1175335 {
```

Comment:
So as per our discussion, I would put this workaround in `lib/registration.pm` as of now. But we still might think of having separate file for it. It's also not that important, as we will have to revert this change, once bug fix is there.

More important thing not to miss is to add `soft_failure` as we don't add any visible reference to the bug.

Reply:
Ok. i tried to think something but i think it is not worth the time to decide one or the other for now. Maybe we can discuss it outside of a PR. i simplified it